scripts/generate_version_docs.py

```diff
@@ -109,5 +109,11 @@ def get_current_versions():
   utils.gsutil_cp(config["version_master_file"], config["doc_bucket"], copy_to_remote=False)
   return utils.read_json_file(config["version_master_file"])
 
+def get_tool_label(tool):
+  if tool == 'gatk':
```

Comment:
I generally like to avoid multiple return statements wherever possible, consider the syntax 
`return tool.upper() if tool == 'gatk' else tool.capitalize()`

Reply:
Thanks! My python intuition is lacking